package.json

```diff
@@ -27,12 +27,26 @@
     "url": "https://github.com/lob/hapi-bookshelf-total-count/issues"
   },
   "homepage": "https://github.com/lob/hapi-bookshelf-total-count#readme",
+  "peerDependencies": {
+    "bookshelf": "0.9.x",
+    "hapi": "11.x"
+  },
```

Comment:
Yup, it would. It might be a little more consistent to follow npm's semver syntax of `^0.x.x`, but I think they're functionally the same thing :+1: